src/app/identity.service.ts

```diff
@@ -33,7 +33,7 @@ import {
   TransactionMetadataCreateNFT,
   TransactionMetadataDAOCoin,
   TransactionMetadataTransferDAOCoin,
-  TransactionSpendingLimit
```

Comment:
Hmm, I wonder why this was here before and wasn't used?

Reply:
Diamondhands made some changes to use a hex string instead of transaction spending limits in this file and we didn't remove the imports.